ci/arm/gitlab-ci.yml

```diff
@@ -0,0 +1,10 @@
+linux-arm64:
```

Comment:
Why is this in an "arm" subdirectory? Doesn't the file have to exist in the root of the repository? (or maybe in a .ci directory? I'm not sure)

Reply:
> Doesn't the file have to exist in the root of the repository? (or maybe in a .ci directory? I'm not sure)

There is a gitlab setting to use a differn't path.